config/config.go

```diff
@@ -53,6 +53,42 @@ type SourcesApiConfig struct {
 	TenantTranslatorUrl       string
 }
 
+//String_Integrate() returns a string that shows the settings in which the pod is running in
```

Comment:
I think you forgot to update the comment:

```diff
- //String_Integrate() returns a string that shows the settings in which the pod is running in
+// String() returns a string that shows the settings in which the pod is running in
```

Other than that, the PR looks good to me :+1:  Good job :+1:

Reply:
<img width="505" alt="Screenshot 2022-07-14 at 19 26 47" src="https://user-images.githubusercontent.com/14937244/179045557-5f8ba68a-8878-4f31-aa86-5d63b814dc2e.png">